src/settings/tabenvironment.cpp

```diff
@@ -0,0 +1,99 @@
+#include "tabenvironment.h"
+
+#include <QDesktopServices>
+#include <QVBoxLayout>
+#include <QSettings>
+#include <QPushButton>
+#include <QComboBox>
+#include <QLabel>
+#include <QDebug>
```

Comment:
remove

Reply:
compiler removing #include <QDebug> in Release mode